ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/state/beaconstate/versions/bellatrix/BeaconStateSchemaBellatrix.java

```diff
@@ -36,7 +36,7 @@
 public class BeaconStateSchemaBellatrix
     extends AbstractBeaconStateSchema<BeaconStateBellatrix, MutableBeaconStateBellatrix> {
 
-  private static final int LATEST_EXECUTION_PAYLOAD_HEADER_FIELD_INDEX = 24;
+  public static final int LATEST_EXECUTION_PAYLOAD_HEADER_FIELD_INDEX = 24;
```

Comment:
I think I'd keep this private and just define it again in Capella.  It happens to be the same but since they're completely separate SszSchemas I think it's worth the constant being separate.  I'm not strongly opinionated about it though so feel free to disagree. :)

Reply:
i was referencing it to get the next offset...